controller/lib/core/sensors.h

```diff
@@ -61,4 +62,8 @@ VolumetricFlow get_volumetric_outflow();
  */
 VolumetricFlow pressure_delta_to_flow(Pressure delta);
 
+// get the sensor readings (patient pressure, volumetric flow and tidal
+// volume) from the sensors
+SensorReadings get_sensor_readings();
```

Comment:
I needed those to be able to run the unit tests, and I believe the math in there needs to be unit tested!
Is there another way?

Reply:
Can we unit test get_sensor_readings() instead? It returns all of the data that was in those three methods, right?

Since get_patient_pressure() etc are not called directly by production code, unit testing them doesn't give us as much confidence that the thing we *actually* call is correct.

Reply:
Will do in a separate PR, this looks a bit much for now...